@viewsort/viewsort.m

```diff
@@ -77,7 +77,21 @@
     cd(Args.HMMDir);
     data.Noise = 1/(hdf5read(Args.HMMFile,Args.HMMNoise));
     cd(cwd);
-             
+    
+    if (isfield(l,'mlseq') > 0)
+        % get and plot ISI coefficient of variation for waveform
+        spikeSort = l.mlseq;
+        for ind = 1:sf1
```

Comment:
You should allocate memory for data.coeffV_ISI before entering the loop by doing:
data.coeffV_ISI = zeros(sf1,1);